original: fullkey: fix “serie”

lucaswerkmeister opened the pull request:
“series” is the singular and plural; “serie” doesn’t exist as an English word, as far as I know. (enwiktionary knows it, but only as an obsolete version of “series”.)

lucaswerkmeister committed:
original: fullkey: fix “serie”

“series” is the singular and plural; “serie” doesn’t exist as an English
word, as far as I know. (enwiktionary knows it, but only as an obsolete
version of “series”.)

maxlath commented:
[half-joking] can't we agree that [English plural needs to be fixed](https://pbs.twimg.com/media/FKhe_QNVQAUXtvU?format=jpg&name=small)? :D

lucaswerkmeister commented:
Why does that image misspell _ghoti_? :P

maxlath committed:
replace "serie" by "series" in keys too

has that's the convention in the fullkey file to have the key and value
be the same in the English version, with the notable exception of the "serie" key itself

associated client repo commit: https://github.com/inventaire/inventaire-client/commit/14569f9
